Kickstarter-iOS/Views/PledgeCTAContainerView.swift

```diff
@@ -143,26 +157,48 @@ final class PledgeCTAContainerView: UIView {
       |> ksr_addSubviewToParent()
       |> ksr_constrainViewToEdgesInParent()
 
-    _ = (self.activityIndicator, self)
+    _ = (self.activityIndicator, self.activityIndicatorContainerView)
       |> ksr_addSubviewToParent()
 
     _ = ([self.titleLabel, self.subtitleLabel], self.titleAndSubtitleStackView)
       |> ksr_addArrangedSubviewsToStackView()
 
-    _ = ([self.titleAndSubtitleStackView, self.spacer, self.pledgeCTAButton], self.rootStackView)
+    _ = (
+      [
+        self.titleAndSubtitleStackView,
+        self.spacer,
+        self.pledgeCTAButton,
+        self.pledgeRetryButton,
+        self.activityIndicatorContainerView
+      ],
+      self.rootStackView
+    )
       |> ksr_addArrangedSubviewsToStackView()
   }
 
   private func setupConstraints() {
     NSLayoutConstraint.activate([
-      self.activityIndicator.heightAnchor.constraint(equalToConstant: Layout.ActivityIndicator.height),
-      self.activityIndicator.widthAnchor.constraint(equalTo: self.activityIndicator.heightAnchor),
-      self.activityIndicator.centerXAnchor.constraint(equalTo: self.layoutMarginsGuide.centerXAnchor),
-      self.activityIndicator.centerYAnchor.constraint(equalTo: self.layoutMarginsGuide.centerYAnchor),
+      // swiftlint:disable line_length
+      self.activityIndicator.centerXAnchor.constraint(equalTo: self.activityIndicatorContainerView.centerXAnchor),
+      self.activityIndicator.centerYAnchor.constraint(equalTo: self.activityIndicatorContainerView.centerYAnchor),
+      self.activityIndicatorContainerView.heightAnchor.constraint(greaterThanOrEqualToConstant: Layout.Button.minHeight),
+      self.activityIndicatorContainerView.widthAnchor.constraint(equalTo: self.layoutMarginsGuide.widthAnchor),
+      // swiftlint:enable line_length
       self.pledgeCTAButton.heightAnchor.constraint(greaterThanOrEqualToConstant: Layout.Button.minHeight),
-      self.pledgeCTAButton.widthAnchor.constraint(greaterThanOrEqualToConstant: Layout.Button.minWidth)
+      self.pledgeCTAButton.widthAnchor.constraint(greaterThanOrEqualToConstant: Layout.Button.minWidth),
+      self.pledgeRetryButton.heightAnchor.constraint(greaterThanOrEqualToConstant: Layout.Button.minHeight),
+      self.pledgeRetryButton.widthAnchor.constraint(greaterThanOrEqualToConstant: Layout.Button.minWidth)
```

Comment:
This constraint is creating a warning, do we need it for this type of button?

<img width="834" alt="image" src="https://user-images.githubusercontent.com/3735375/63285741-52537080-c26b-11e9-85b7-0ae0afc45601.png">